tests/rising_bubble_ls.prm

```diff
@@ -23,7 +23,7 @@ subsection Two phase
   set density difference   = -0.9
   set viscosity            = 0.01
   set viscosity difference = -0.009
-  set surface tension      = 0.0245
```

Comment:
Is this change on purpose?

Reply:
Nope. I was only playing around. I'll fix it in a couple minutes.